item.rb

```diff
@@ -0,0 +1,22 @@
+class Item
+  attr_accessor :publish_date
+  attr_reader :archived, :id, :genre, :label, :author
+
+  def initialize(publish_date, archived: false, id: nill)
+    super()
+    @id = id || Random.rand(1..1000)
+    @archived = archived
+    @publish_date = publish_date
+    @label = nill
```

Comment:
By default, @label will be nil so you can just leave it as `@label`

Reply:
Just fixed it. :+1: